process.py

```diff
@@ -190,7 +190,7 @@ def remove_groups(api, users, groups_to_remove_from):
         {
             "paging": False,
             "filter": "name:in:[%s]" % ",".join(groups_to_remove_from),
-            "fields": ("id,name,users"),
+            "fields": ("*"),
```

Comment:
[very minor] We can remove these parentheses, it looks weird (I guess there was some `("ab" "cd")` string concatenation at some point).